products/tarantool.md

```diff
@@ -0,0 +1,70 @@
+---
+permalink: /tarantool
+layout: post
+title: Tarantool
+command: $ tarantool --version
+releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
+changelogTemplate: |
+  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
+category: db
+iconSlug: NA
+eolColumn: Support Status
+releaseDateColumn: false
+sortReleasesBy: 'releaseCycle'
+releases:
+  - releaseCycle: "2.8"
+    release: 2021-12-22
+    eol: false
+    latest: "2.8.3"
+
+  - releaseCycle: "2.7"
+    release: 2021-08-19
+    eol: 2021-08-19
+    latest: "2.7.3"
+
+  - releaseCycle: "2.6"
+    release: 2021-04-21
+    eol: 2021-04-21
+    latest: "2.6.3"
+
+  - releaseCycle: "2.5"
+    release: 2020-12-30
+    eol: 2020-12-30
+    latest: "2.5.3"
+
+  - releaseCycle: "2.4"
+    release: 2020-10-22
+    eol: 2020-10-22
+    latest: "2.4.3"
+
+  - releaseCycle: "2.3"
+    release: 2020-07-17
+    eol: 2020-07-17
+    latest: "2.3.3"
+
+  - releaseCycle: "2.2"
+    release: 2019-12-31
+    eol: 2020-04-20
+    latest: "2.2.2"
+
+  - releaseCycle: "1.10"
+    release: 2021-12-22
+    eol: false
+    lts: true
+    latest: "1.10.12"
+
+---
+
+> [Tarantool](https://www.tarantool.io/) is an open source (BSD-2 clause licensed), in-memory computing platform consisting of a database and an application server. It includes a heavily optimized Lua interpreter, cooperative multitasking and non-blocking IO, persistent queues, sharding, and MySQL/PostgreSQL support.
+
+The Tarantool release policy is changing to become more clear and intuitive. The new policy uses a [SemVer-like](https://semver.org/) versioning format, and introduces a new version lifecycle with more long-time support series. This document explains the new release policy, versioning rules, and [release series](https://www.tarantool.io/en/doc/latest/release/policy/#term-Release-series) lifecycle.
+
+The new release policy replaces the [legacy policy](https://www.tarantool.io/en/doc/latest/release/legacy-policy/) for:
+
+- The `2.x.y` series since the `2.10.0` release. Development for this new release starts with version `2.10.0-beta1`.
+- The future `3.0.0` series.
```

Comment:
I would say it shorter:

> ...for `2.x.y` series since `2.10.0` release and all future series (`3.x.y` and so on).

Maybe we have too much emphasis on this fact now.

Reply:
Done.

```diff
@@ -59,10 +62,7 @@ releases:
 
 The Tarantool release policy is changing to become more clear and intuitive. The new policy uses a [SemVer-like](https://semver.org/) versioning format, and introduces a new version lifecycle with more long-time support series. This document explains the new release policy, versioning rules, and [release series](https://www.tarantool.io/en/doc/latest/release/policy/#term-Release-series) lifecycle.
 
-The new release policy replaces the [legacy policy](https://www.tarantool.io/en/doc/latest/release/legacy-policy/) for:
-
-- The `2.x.y` series since the `2.10.0` release. Development for this new release starts with version `2.10.0-beta1`.
-- The future `3.0.0` series.
+The new release policy replaces the [legacy policy](https://www.tarantool.io/en/doc/latest/release/legacy-policy/) for 2.x.y series since 2.10.0 release and all future series (3.x.y and so on).
 
 Here are the most significant changes from the legacy release policy:
 
```